docs/_plugins/template.md

```diff
@@ -0,0 +1,12 @@
+---
+name: requried
+category: requried (same name as directory)
+repo: requried
+author: requried
+author-url:
+demo: 
+compatible-v0:
+compatible-v1: true
+---
+
+Describe here your plugin. Keep it short and check out the https://github.com/Leaflet/Leaflet/blob/master/PLUGIN-GUIDE.md.
```

Comment:
You could add a markdown link perhaps?

```
[plugin guide](https://github.com/Leaflet/Leaflet/blob/master/PLUGIN-GUIDE.md)
```

Reply:
Or mention that markdown can be used if needed.